shared/src/business/utilities/getDescriptionDisplay.test.js

```diff
@@ -0,0 +1,84 @@
+const { getDescriptionDisplay } = require('./getDescriptionDisplay');
+
+describe('getDescriptionDisplay', () => {
+  const documentTitle = 'doc title';
+  const documentType = 'doc type';
+  const description = 'doc description';
+
+  let documentMetaData = {
+    documentTitle,
+  };
+
+  it('returns descriptionDisplay as documentTitle with no added information', () => {
+    const expected = documentTitle;
+    const result = getDescriptionDisplay(documentMetaData);
+
+    expect(result).toEqual(expected);
+  });
+
+  it('returns descriptionDisplay as documentType with no added information', () => {
+    const expected = 'doc type';
+    documentMetaData = {
+      ...documentMetaData,
+      documentTitle: undefined,
+      documentType,
+    };
+    const result = getDescriptionDisplay(documentMetaData);
+
+    expect(result).toEqual(expected);
```

Comment:
Seems like a different approach than other tests in the codebase by using `expected` and not simply using that value in the expect at the bottom.

```

    const result = getDescriptionDisplay({
      documentType,
    });

    expect(result).toEqual('doc type');
```
this could be easier to read and understand.

Reply:
We do actually see this pattern all over our tests, but I agree it's easier to read/understand if it's not unnecessarily extracted out.

Reply:
You're right. There is a mix of patterns in the tests, and we do see expected. It just seems like that's used for more involved checks of objects rather than strings.

Reply:
@mmarcotte Made the changes as suggested. Thanks for bringing it up